server/gameData.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/jak103/uno/db"
+)
+
+// HandInfo is a JSON struct for holding summary information about each players hand for a given game.
+type HandInfo struct {
+	PlayerName string `json:"name"`
+	CardCount  int    `json:"numCards"`
+}
+
+func getCardCount(gameID string) (string, error) {
```

Comment:
Let's not create a new file for a single function. Move this over to uno.go.

Reply:
Done.